feature/subscriber-attributes/src/main/java/com/revenuecat/purchases/subscriberattributes/SubscriberAttributesManager.kt

```diff
@@ -51,7 +52,7 @@ class SubscriberAttributesManager(
         val unsyncedStoredAttributesForAllUsers =
             deviceCache.getUnsyncedSubscriberAttributes()
         if (unsyncedStoredAttributesForAllUsers.isEmpty()) {
-            debugLog("No subscriber attributes to synchronize.")
+            debugLog(AttributionStrings.NO_SUBSCRIBER_ATTRIBUTES_TO_SYNCHRONIZE)
```

Comment:
Nice thing about this is that this gets inlined during compile time. If you look at the bytecode, this looks like this:

`LogUtilsKt.debugLog("No subscriber attributes to synchronize.");`